src/graphnet/models/standard_model.py

```diff
@@ -94,22 +180,14 @@ def configure_optimizers(self) -> Dict[str, Any]:
             )
         return config
 
-    def forward(
-        self, data: Union[Data, List[Data]]
-    ) -> List[Union[Tensor, Data]]:
+    def forward(self, data: Data) -> List[Union[Tensor, Data]]:
```

Comment:
Looks like you've removed the functionality introduced in #610. Was this intentional? If so, could you elaborate on why?

Reply:
@RasmusOrsoe Good catch, it would seem I was doing the refactor from an outdated branch prior to #610 . I've now added the functionality from #610 to the refactor.